hw/ip/ascon/data/ascon.hjson

```diff
@@ -616,27 +616,11 @@
           name: "ALERT_RECOV_CTRL_UPDATE_ERR",
           resval: "0",
           desc:  '''
-            An update error has not occurred (0) or has occurred (1) in the shadowed Control Register.
-            Ascon operation needs to be restarted by re-writing the Control Register.
+            An update error has not occurred (0) or has occurred (1) in one of the shadowed Control
+            Register. Ascon operation needs to be restarted by re-writing the Control Register.
```

Comment:
Nit: Control Register -> control registers because now two registers feed into this update error.